lib/sendgrid/helpers/helpers.rb

```diff
@@ -0,0 +1,24 @@
+module SendGrid
+  module Helpers
+
+    # this method name is not accurate.  This method returns a hash of the non-nil instance variables
+    # TODO: update all uses to use :to_hash
+
+    def to_json
+      return nil if nil?
```

Comment:
Since none of the `helpers` define a `nil?` method, I don't think this will ever be used.